specification/logs/semantic_conventions/events/browser/resource.md

```diff
@@ -0,0 +1,38 @@
+# Navigation
+
+**Status**: [Experimental](../../../../document-status.md)
+
+**type:** `resource`
+
+**Description**: Represents a resource observed by the browser.
+
+The following attributes MUST be set with these predefined values:
+
+<!-- semconv browser -->
+| Attribute  | Type | Value  | Requirement Level |
+|---|---|---|---|
+| `event.name` | string | `resource` | Required |
+| `event.domain` | string | `browser` | Required |
+<!-- endsemconv -->
+
+Additional attributes are mapped from the [Resource Timing API](https://www.w3.org/TR/resource-timing-2/#sec-performanceresourcetiming).
+
+This list is not exhaustive and is listed as an example. All numeric and string values thar are provided by the Resource Timing API SHOULD be included.
```

Comment:
We should call out that this should be included the `event.data` attribute.

Reply:
Should all events always add their attributes in `event.data`? If yes, then I think we should add it to the spec for the `event.*` attributes, which is currently proposed in https://github.com/open-telemetry/opentelemetry-specification/pull/2676.

Reply:
Its also a case that as part of the "Nested Attribute" support I included wording about effectively exporters needing to use a strategy to handle when they receive nested attributes, which includes the default (already mentioned as an option) using JSON.stringify()